### ISSUE TITLE
Loss stays same throught training

### ISSUE THREAD

**Issue by turinaf** [musikalkemist/pytorchforaudio#9]
I trained this urban sound classifier for 10 epochs, but there one problem
The loss stays extactly same except for 1st epochs. What could be the reason? 

![image](https://github.com/musikalkemist/pytorchforaudio/assets/40395348/a0669925-7126-45c4-a583-6bf8ff273307)

**Comment by pgq18** [comment]
> I trained this urban sound classifier for 10 epochs, but there one problem The loss stays extactly same except for 1st epochs. What could be the reason?
> 
> ![image](https://private-user-images.githubusercontent.com/40395348/300345562-a0669925-7126-45c4-a583-6bf8ff273307.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5MDU3NjAsIm5iZiI6MTcxMDkwNTQ2MCwicGF0aCI6Ii80MDM5NTM0OC8zMDAzNDU1NjItYTA2Njk5MjUtNzEyNi00NWM0LWE1ODMtNmJmOGZmMjczMzA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIwVDAzMzEwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwN2FmMmZkYzg4NDcyNWJhZjQwMmJhMGFhZTExYzY3NjQwNTgxMmFjMjA0NjI4Mjk1MTFkZDExYTc0MDdlOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oSYhR3pTGh70XKUDFAsbeSfp-YyG4PG3mdabNZgJC4o)

Hello. I had met the same problem with yours. Have you solved the problem?

**Comment by turinaf** [comment]
> > I trained this urban sound classifier for 10 epochs, but there one problem The loss stays extactly same except for 1st epochs. What could be the reason?
> > ![image](https://private-user-images.githubusercontent.com/40395348/300345562-a0669925-7126-45c4-a583-6bf8ff273307.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5MDU3NjAsIm5iZiI6MTcxMDkwNTQ2MCwicGF0aCI6Ii80MDM5NTM0OC8zMDAzNDU1NjItYTA2Njk5MjUtNzEyNi00NWM0LWE1ODMtNmJmOGZmMjczMzA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIwVDAzMzEwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwN2FmMmZkYzg4NDcyNWJhZjQwMmJhMGFhZTExYzY3NjQwNTgxMmFjMjA0NjI4Mjk1MTFkZDExYTc0MDdlOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oSYhR3pTGh70XKUDFAsbeSfp-YyG4PG3mdabNZgJC4o)
> 
> Hello. I had met the same problem with yours. Have you solved the problem?

Try changing cnn architecture. I tried completely different one and loss changes starts to decrease


![image](https://github.com/musikalkemist/pytorchforaudio/assets/40395348/4f598206-6ce8-4ea8-bc96-e9b40c8f49cc)

**Comment by pgq18** [comment]
Thanks for your answer! I solved the problem after looking through the comments under the Author's lessons on YouTube. The viewers had the same problems. It seems to be the conflict between the loss function CrossEntropyLoss and the softmax layer. After I removed the softmax layer, training became normal and the model works well.
![image](https://github.com/musikalkemist/pytorchforaudio/assets/126139161/2925b963-fe12-49ed-801d-8bfbb941710f)
